### PR TITLE
feat: add query and profile log config

### PR DESCRIPTION
**Pull request by hantmac**

### CODE REVIEW

File: charts/databend-query/values.yaml [hunk 1]
```diff
@@ -100,6 +100,20 @@ config:
     stderr:
       enabled: true
       level: "WARN"
+    query:
+      enabled: false
+      otlp_endpoint: ""
+      otlp_protocol: "http"
+      otlp_labels:
+        tenant: ""
```
Comment (everpcpc) [inline charts/databend-query/values.yaml:108]: labels should be optional.

Comment (hantmac) [reply charts/databend-query/values.yaml]: Fixed.

File: charts/databend-query/templates/configmap.yaml [hunk 2]
```diff
@@ -53,6 +53,28 @@ data:
       [log.stderr]
         on = {{ .Values.config.log.stderr.enabled }}
         level = {{ .Values.config.log.stderr.level | default "WARN" | quote }}
+      [log.query]
+        on = {{ .Values.config.log.query.enabled }}
+        otlp_endpoint = {{ .Values.config.log.query.otlp_endpoint | quote }}
+        otlp_protocol = {{ .Values.config.log.query.otlp_protocol | default "http" | quote }}
+        [log.query.otlp_labels]
```
Comment (everpcpc) [inline charts/databend-query/templates/configmap.yaml:60]: `{{ with .Values.config.log.profile.otlp_labels }}` ?

Comment (hantmac) [reply charts/databend-query/templates/configmap.yaml]: Done.

File: charts/databend-query/templates/configmap.yaml [hunk 3]
```diff
@@ -53,6 +53,32 @@ data:
       [log.stderr]
         on = {{ .Values.config.log.stderr.enabled }}
         level = {{ .Values.config.log.stderr.level | default "WARN" | quote }}
+      [log.query]
+        on = {{ .Values.config.log.query.enabled }}
+        otlp_endpoint = {{ .Values.config.log.query.otlp_endpoint | quote }}
+        otlp_protocol = {{ .Values.config.log.query.otlp_protocol | default "http" | quote }}
+      {{- with .Values.config.log.query.otlp_labels }}
+      [log.query.otlp_labels]
+        {{- if .tenant }}
```
Comment (everpcpc) [inline charts/databend-query/templates/configmap.yaml:62]: We should not specify `tenant` and `warehouse` here, just assuming labels is a general map.

File: .idea/workspace.xml [hunk 4]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (everpcpc) [inline .idea/workspace.xml:1]: git ignore?

Comment (hantmac) [reply .idea/workspace.xml]: Sorry, forgot it.

**Review by everpcpc — APPROVED** [review]

**Pull request merged**